cmd/lncli/walletrpc_active.go

```diff
@@ -1193,6 +1195,178 @@ func listAddresses(ctx *cli.Context) error {
 	return nil
 }
 
+var signMessageWithAddrCommand = cli.Command{
+	Name:  "signmessagewithaddr",
```

Comment:
Since this will be a sub command of `lncli wallet addresses`, can it just be `signmessage` and `verifymessage`?

Reply:
good idea